en-US/Players_Guide/06_Spellcasting/Spell_Lists/Warlock_Spells.md

```diff
@@ -3,8 +3,9 @@
 
 ##### Cantrips (0 Level)
 
-- Chill Touch
-- Eldritch Blast
+- [Chill Touch](#Chill_Touch_chill_touch)
+- [Eldritch Blast](#Eldritch_Blast_eldritch_blast)
+- [Friends](#Friends_friends)
```

Comment:
This seems to be a new A5e spell that was not yet in the spell list. According to the A5e spell document however, it is a Warlock spell, so I added it here.